app/pb_kits/playbook/pb_selectable_list/_selectable_list.scss

```diff
@@ -0,0 +1,9 @@
+@import "../tokens/colors";
+
+[class^=pb_selectable_list]{
+  .pb_item_kit {
+    &:hover {
+      background-color: rgb(249,251,255);
```

Comment:
This is worrysome. We really want to try and use `tokens` if we can for these scenarios: https://playbook.powerapp.cloud/utilities

Reply:
Changed it to use the token